ArduCopter/AP_Arming.cpp

```diff
@@ -351,19 +351,19 @@ void AP_Arming_Copter::pre_arm_rc_checks(const bool display_failure)
         // check if radio has been calibrated
         if (!channel->min_max_configured()) {
             if (display_failure) {
-                copter.gcs_send_text_fmt(MAV_SEVERITY_CRITICAL,"PreArm: RC %s not configured", channel_name);
+                copter.gcs().send_text(MAV_SEVERITY_CRITICAL,"PreArm: RC %s not configured", channel_name);
```

Comment:
Remove the copter part to make it consistent with other places. There are more below.